docs/user/getting_started.rst

```diff
@@ -91,15 +91,9 @@ referred to as ``$WECSIM`` (e.g. ``C:/User/Documents/GitHub/WEC-Sim``).
 Step 1. Add WEC-Sim to MATLAB Path
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Open the ``$WECSIM/source/wecSimStartup.m`` file.
+Open the ``$WECSIM folder and run the "activate_wecsim.m" script.
```

Comment:
There is no mention of creating a `work` folder in the current directory. Therefore, if this pull request is accepted then the folder will need to be added so that users do not have to create themselves. Or the documentation will need a line explaining that a `work` folder needs to be added.

Reply:
I automated this by the most recent commit.